config/settings.py

```diff
@@ -63,6 +63,7 @@
 background_colour = [26, 26, 27]
 background_opacity = 0.5
 background_volume = 0.5
+# commentstyle = ["reddit", "text"]
```

Comment:
If Reddit comment screenshotting breaks, you can always fall back to `commentstyle = "text"` as a workaround until the reddit style screenshotting logic is fixed.